src/insert/mobile-crossword-banner.ts

```diff
@@ -24,13 +24,13 @@ export const init = (): Promise<void> => {
 		window.guardian.config.tests?.crosswordMobileBannerVariant ===
 		'variant';
 
-	if (window.guardian.config.isDotcomRendering || !isInABTest) {
+	if (window.guardian.config.isDotcomRendering) {
 		return Promise.resolve();
 	}
 
-	const anchorSelector = window.guardian.config.page.commentable
-		? '.crossword__container__below-controls + *'
-		: '.content-footer > :first-child';
+	const anchorSelector = isInABTest
```

Comment:
is the `window.guardian.config.page.commentable` check not needed anymore?

Reply:
We used to use this for the `high-merch` slot (which would add a slot after the comments, having the footer as a fallback). For this advert on the crosswords page, we don't use the comments section to place an advert.

Reply:
Ah I see!